girder/constants.py

```diff
@@ -195,6 +195,12 @@ class TokenScope:
     ANONYMOUS_SESSION = 'core.anonymous_session'
     USER_AUTH = 'core.user_auth'
     TEMPORARY_USER_AUTH = 'core.user_auth.temporary'
+    PLUGINS_ENABLED_READ = 'core.plugins.read'
```

Comment:
Good points. I can tolerate some deprecated variable identifiers lingering in our internal code for the sake of plugins.